<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QDialog" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Quiz App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #F0F8FF, stop:1 #DFF1FF);
  font-family: 'Segoe UI', Arial, sans-serif;
}
QLabel {
  color: #0D3D56;
}
QPushButton {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #4C9CC0, stop:1 #0D3D56);
  color: white;
  border: none;
  border-radius: 12px;
  padding: 12px 20px;
  font-weight: bold;
  font-size: 12px;
  text-align: center;
}
QPushButton:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #5FAECE, stop:1 #327A9F);
  transform: translateY(-2px);
}
QPushButton:pressed {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #0D3D56, stop:1 #0A2D40);
  transform: translateY(0px);
}
.card-container {
  background: rgba(255, 255, 255, 0.7);
  border-radius: 16px;
  padding: 20px;
  margin: 10px;
  border: 1px solid rgba(13, 61, 86, 0.1);
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_Main">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>400</width>
     <height>897</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="headerFrame">
      <property name="styleSheet">
       <string notr="true">QFrame {
  background: rgba(255, 255, 255, 0.8);
  border-radius: 20px;
  padding: 20px;
  border: 1px solid rgba(13, 61, 86, 0.1);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="headerLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="welcomeContainerLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="welcomeLabel">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>16</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #5A7A8A; font-weight: 400;</string>
           </property>
           <property name="text">
            <string>Welcome back,</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="UsernameTitle">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #0D3D56; font-weight: 700;</string>
           </property>
           <property name="text">
            <string>User</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="actionsFrame">
      <property name="styleSheet">
       <string notr="true">QFrame {
  background: rgba(255, 255, 255, 0.6);
  border-radius: 20px;
  padding: 15px;
  border: 1px solid rgba(13, 61, 86, 0.1);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="actionsLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>25</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="QLabel" name="actionsLabel">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #0D3D56; margin-bottom: 10px;</string>
         </property>
         <property name="text">
          <string>Quick Actions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UploadAndGenerateButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #5FAECE, stop:1 #327A9F);
  border-radius: 15px;
  padding: 15px 25px;
  font-size: 13px;
  font-weight: bold;
}
QPushButton:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #6BB8D6, stop:1 #3A89AD);
  box-shadow: 0 4px 8px rgba(0,0,0,0.1);
}</string>
         </property>
         <property name="text">
          <string>ðŸ“„ Upload PDF &amp; Generate Quiz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UploadButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #4C9CC0, stop:1 #1F5275);
  border-radius: 15px;
  padding: 15px 25px;
  font-size: 13px;
  font-weight: bold;
}
QPushButton:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #5FAECE, stop:1 #286E94);
  box-shadow: 0 4px 8px rgba(0,0,0,0.1);
}</string>
         </property>
         <property name="text">
          <string>ðŸ“¤ Upload PDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GenerateFromOldButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #3A89AD, stop:1 #0D3D56);
  border-radius: 15px;
  padding: 15px 25px;
  font-size: 13px;
  font-weight: bold;
}
QPushButton:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #4C9CC0, stop:1 #1A4A63);
  box-shadow: 0 4px 8px rgba(0,0,0,0.1);
}</string>
         </property>
         <property name="text">
          <string>ðŸ”„ Generate Quiz From Previous PDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PassQuizButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #286E94, stop:1 #0A2D40);
  border-radius: 15px;
  padding: 15px 25px;
  font-size: 13px;
  font-weight: bold;
}
QPushButton:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #327A9F, stop:1 #0D3D56);
  box-shadow: 0 4px 8px rgba(0,0,0,0.1);
}</string>
         </property>
         <property name="text">
          <string>ðŸŽ¯ Take a Quiz</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="bottomSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
